website/docs/install-config/configuration/configuration.mdx

```diff
@@ -111,21 +142,24 @@ The same PostgreSQL settings as described above are used for each read replica.
 - `AUTHENTIK_POSTGRESQL__READ_REPLICAS__0__CONN_HEALTH_CHECKS`
 - `AUTHENTIK_POSTGRESQL__READ_REPLICAS__0__DISABLE_SERVER_SIDE_CURSORS`
 
-### Using a PostgreSQL connection pooler (PgBouncer or PgPool)
+### Using a PostgreSQL Connection Pooler
+
+When your PostgreSQL databases are running behind a connection pooler (like PgBouncer or PgPool), you need to adjust several settings to ensure compatibility:
 
-When your PostgreSQL database(s) are running behind a connection pooler, like PgBouncer or PgPool, two settings need to be overridden:
+- `AUTHENTIK_POSTGRESQL__CONN_MAX_AGE`:
 
-- `AUTHENTIK_POSTGRESQL__CONN_MAX_AGE`
+    A connection pooler running in session pool mode (which is the default for PgBouncer) can be incompatible with unlimited persistent connections (`null` setting). When the connection from the pooler to the database is dropped, the pooler will wait for the client to disconnect before releasing the connection. However, this will **never** happen as authentik keeps the connection indefinitely.
```

Comment:
```suggestion
    A connection pooler running in session pool mode (which is the default for PgBouncer) can be incompatible with unlimited persistent connections (`null` setting). When the connection from the pooler to the database is dropped, the pooler will wait for the client to disconnect before releasing the connection. However, this will *never* happen as authentik keeps the connection indefinitely.
```

Reply:
for emphasis we want italics...